### PR TITLE
Update binance endpoint to v3

### PR DESCRIPTION
**Pull request by halink0803**

### CODE REVIEW

File: exchange/huobi/huobi_endpoint.go [hunk 1]
```diff
@@ -361,8 +361,6 @@ func (ep *Endpoint) GetDepositAddress(asset string) (exchange.HuobiDepositAddres
 			err = fmt.Errorf("get Huobi deposit address failed: %s", result.Message)
 		}
 	}
-	// log response for debugging
```
Comment (secmask) [inline exchange/huobi/huobi_endpoint.go]: this should be put in a separate commit.

Comment (halink0803) [reply exchange/huobi/huobi_endpoint.go]: Do you mean separate PR or just separate commit?

Comment (secmask) [reply exchange/huobi/huobi_endpoint.go]: this is small change, so a separate commit is acceptable.
